docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,131 @@
+---
+slug: /dev/project-setup
+---
+
+# Paper Project Setup
+
+As the Paper team primarily uses [IntelliJ IDEA](https://www.jetbrains.com/idea/), this guide will be focused on that IDE. 
+However, the steps below should be applicable to other IDEs as well, with some minor changes.
+
+
+### Creating a new project
+
+Open your IDE and select the option to create a new project. 
+In Intellij, you will get the option to select the type of project you want to create. 
+Select `Gradle - Kotlin DSL` and click `Create`.
+
+You will land into the `build.gradle.kts` file where you can add your dependencies.
+
+### Adding Paper as a dependency
+
+To add Paper as a dependency, you will need to add the Paper repository to your `build.gradle.kts` file as well as the dependency itself.
+
+```kotlin
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://repo.papermc.io/repository/maven-public/")
+    }
+}
+
+dependencies {
+    compileOnly("io.papermc.paper:paper-api:1.19.2-R0.1-SNAPSHOT")
+}
+
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+```
+
+:::info
+
+To see the versions of Paper you want to use, visit our [Downloads](https://www.papermc.io/downloads) page.
+
+:::
+
+### Setting up the `src` directory
+
+To set up the `src` directory, you will need to create a new directory called `src` and then create a new directory called `main` inside of it.
+Inside of `main`, create two new directories called `java` and `resources`.
+
+It should look like this:
+
+```
+...
+src
+└── main
+    ├── java
+    └── resources
+...
+```
+
+### Setting up the `java` directory
+
+You will place your Java source files in the `java` directory. You first need to create some packages to organize your code.
+For this example, we will create three packages called `io.papermc.testplugin` and then create a class called `ExamplePlugin` inside of it.
+
+```
+...
+src
+└── main
+    ├── java
+    │   └── io
+    │       └── papermc
+    │           └── testplugin
+    │               └── ExamplePlugin.java
+    └── resources
```

Comment:
It would make sense to apply this change to the rest of your PR (2)

Reply:
Yep, Done!